iceoryx_examples/icedelivery/iox_publisher_typed.cpp

```diff
@@ -39,8 +39,8 @@ int main()
 
     iox::runtime::PoshRuntime::initRuntime("iox-ex-publisher-typed");
 
-    iox::popo::TypedPublisher<RadarObject> typedPublisher({"Radar", "FrontLeft", "Object"});
-    typedPublisher.offer();
+    iox::popo::TypedPublisher<RadarObject> publisher({"Radar", "FrontLeft", "Object"});
```

Comment:
Renaming of `TypedPublisher` into `Publisher` and `UntypedPublisher` in `RawPublisher` will be done in the next PR, right?

Reply:
Yes, this change would propagate through all examples and many tests.